src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -293,8 +305,83 @@ private void initProperties() {
         rpcLoginTimeout = parseToInt(RPC_LOGIN_TIMEOUT.getKey(), rpcLoginTimeout);
     }
 
+    private void initMetadataFromRS() throws Exception {
+
+        List<ObServerAddr> rsList = new ArrayList<ObServerAddr>();
+        for (String serverStr : getRsList().split(":")){
+            ObServerAddr serverAddr = new ObServerAddr();
+            serverAddr.setAddress(serverStr);
+            rsList.add(serverAddr);
+        }
+
+        List<ObServerAddr> servers = new ArrayList<ObServerAddr>();
+        ConcurrentHashMap<ObServerAddr, ObTable> tableRoster = new ConcurrentHashMap<ObServerAddr, ObTable>();
+
+        TableEntryKey rootServerKey = new TableEntryKey(clusterName, tenantName,
+            OCEANBASE_DATABASE, ALL_DUMMY_TABLE);
+
+
+        TableEntry tableEntry = loadTableEntryRandomly(rsList,//
```

Comment:
I think it's better to reuse the logic below, as the code is mainly the same with the original method.